### PR TITLE
v2.0.1

### PR DESCRIPTION
**Pull request by ppetrosh**
- Add refresh token for the smart refrig example
- #13 is being addressed

### CODE REVIEW

**Comment by ppetrosh** [comment]
I double-tested the change. It seems to work for me.

@betzrhodes can you please give it a quick look?

File: examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut [hunk 1]
```diff
@@ -26,7 +26,8 @@
 #require "Rocky.class.nut:1.2.3"
 
 // Web Integration Library
-#require "Salesforce.agent.lib.nut:2.0.0"
+// #require "Salesforce.agent.lib.nut:2.0.0"
```
Comment (betzrhodes) [inline examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut]: May want to switch back to require statement before publishing

File: examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut [hunk 2]
```diff
@@ -194,36 +206,49 @@ class SmartFridgeApplication {
...
 }
 
 // RUNTIME
 // ---------------------------------------------------------------------------------
 
 // SALESFORCE CONSTANTS
 // ----------------------------------------------------------
-const CONSUMER_KEY = "<YOUR_CONSUMER_KEY_HERE>";
-const CONSUMER_SECRET = "<YOUR_CONSUMER_SECRET_HERE>";
+const CONSUMER_KEY = "@{SALESFORCE_CONSUMER_KEY}";
```
Comment (betzrhodes) [inline examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut:250]: add comment about updating these variables with users credentials

**Review by betzrhodes — APPROVED** [review]
Update looks good. There were a couple places in the example that were adjusted for testing that may want to change back before publishing. I've noted them below.

File: examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut (hunk 1 above)
Comment (ppetrosh) [reply examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut]: Fixed.

File: examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut (hunk 2 above)
Comment (ppetrosh) [reply examples/SmartRefrigerator/SmartRefrigerator_ExplorerKit_Salesforce.agent.nut:250]: Fixed

**Pull request merged**